### ISSUE TITLE
Maximum recursion depth exceeded error when list is not closed

### ISSUE THREAD

**Issue by restless** [livingbio/fuzzy-json#113]
`fuzzy_loads` fails with the maximum recursion depth exceeded error for the following (and similar) invalid json strings:

```
# the list is missing the `]`
json_str = """
{"attr1": [{"a": "1"},{"a": "2"}}
"""

fuzzy_loads(json_str)

(...)
JSONDecodeError: Failed to repair JSON: maximum recursion depth exceeded: line 1 column 1 (char 0)

```

in case there are integers instead of strings in the dicts it is:

```
# the list is missing the `]`
json_str = """
{"attr1": [{"a": 1},{"a": 2}}
"""

fuzzy_loads(json_str)

(...)

JSONDecodeError: Failed to repair JSON: Invalid JSON } in state_post_value: line 1 column 1 (char 0)=
```

**Comment by lucemia** [comment]
@restless Thanks for report, the issue has been fixed in 0.0.10 release

**Comment by restless** [comment]
Wow, that was quick. Thank you!
